### DISCUSSION TITLE
New exploit!

### DISCUSSION THREAD

**Discussion by DJshelfmushroom** [theshmunguss1/RamHub#10] [Show and tell]
Steps to create (tested on securly only)
create a google doc
put whatever links you want in it
download it as a pdf
click the link
unblocked!

**Comment by theshmunguss1** [comment]
That's such a simple exploit, you should try to keep it low-key so Securely doesn't patch it. Could you post a video?

**Reply by DJshelfmushroom** [reply]
[Screen recording 2024-04-09 6.24.28 PM.webm](https://github.com/theshmunguss1/RamHub/assets/67295452/31a2a8ce-7c94-49db-af3a-6bd06cc9a484)

**Comment by YesTheProxy** [comment]
Doesn't work on Cisco Umbrella

**Reply by DJshelfmushroom** [reply]
):

**Reply by theshmunguss1** [reply]
Cisco is very secure about their service, it's amazing how proxies even work

**Comment by DJshelfmushroom** [comment]
THEY ALREADY F---ING PATCHED IT WTFFFFFFFF

**Comment by DJshelfmushroom** [comment]
wait no they didn't

**Reply by DJshelfmushroom** [reply]
WAIT YOU HAVE TO DO IT FROM A NEW TAB??? i am so confused

**Reply by theshmunguss1** [reply]
dude is having conversations with himself

**Reply by Breezy20111117** [reply]
How to get the link if it is blocked on securly

**Reply by WiFiGiver** [reply]
This isn't Google bruh

**Comment by DJshelfmushroom** [comment]
Stfu lol

**Reply by theshmunguss1** [reply]
I'm confused, were you trying to reply to someone?

**Reply by DJshelfmushroom** [reply]
yeah it didn't work, i replied to the email notification but it replied to the post

**Comment by DJshelfmushroom** [comment]
well folks, we had a good run.

**Reply by DJshelfmushroom** [reply]
god DAMNIT

**Reply by theshmunguss1** [reply]
I wonder if this implies that Securely is scanning Github for stuff like this?

**Comment by DJshelfmushroom** [comment]
@theshmunguss1 @YesTheProxy GOT IT WORKING AGAIN

**Reply by DJshelfmushroom** [reply]
you have to open memory sucking tabs, I used [adobe express](https://new.express.adobe.com/new?width=1080&height=1920&unit=px&aspectRatioLock=true&category=templates&taskID=standard-size-portrait) but should work with anything

**Reply by DJshelfmushroom** [reply]
lmaooooooooooooooooooooo

**Comment by DJshelfmushroom** [comment]
If it's blocked, it's blocked
